buildinfo/buildinfo.go

```diff
@@ -0,0 +1,31 @@
+package buildinfo
+
+import "fmt"
+
+var (
+	// GitCommit is set by govvv at build time.
+	GitCommit = ""
```

Comment:
went with empty strings over 'n/a' as it was in powergate. i do some logic against these, so empty strings were more helpful. flagging if you did 'n/a' for a critical reason @asutula